src/components/HistoryTable/HistoryTable.tsx

```diff
@@ -52,13 +53,21 @@ const HistoryTable = ({
     }
   }, [processedCommitCount]);
 
+  console.log({ processedCommitCount });
   useEffect(() => {
-    if (selectedCommit && selectedCommit.rowIndex && listRef.current) {
+    if (
+      selectedCommit &&
+      selectedCommit.rowIndex &&
+      listRef.current &&
+      !selectedCommit.visited
```

Comment:
Maybe I am wrong but I still don't understand why this `visited` property is needed.

For example, I changed the hook to this, and the bug doesn't seem to occur anymore:

<img width="739" alt="Screen Shot 2022-04-08 at 10 03 18 AM" src="https://user-images.githubusercontent.com/47064971/162451950-b67e0ef5-c77c-4b4c-b268-573c1643e02d.png">

Which sort of makes me feel like this can be done without the `visited` property, but let me know if there's something I am overlooking